editor/block-mover/test/mover-label.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { getBlockMoverLabel, getMultiBlockMoverLabel } from '../mover-label';
+
+describe( 'block mover', () => {
+	const dir_up = -1,
```

Comment:
Ugh, sorry about that @aduth - In my defence this is my first ever WP contribution! Will read up on the standards and get better acquainted with the way things are done here.

On another note, I see eslint does have camelCase enforcement built in and wonder if we'd benefit from putting it into the `.eslintrc` file? Happy to do another PR if you'd like to add this.